### DISCUSSION TITLE
Is it possible to see the last modification date of different parts of a submission?

### DISCUSSION THREAD

**Discussion by amarsman** [pretalx/pretalx#1315] [Ideas] [answered]
During the Content work for MCH2022 we had multiple times the desire to see the modification date of a specific part of a submission, e.g.:

* After a submission was reviewed the submitter totally changed their submission so the submission and the reviews didn't have any relation any more
* At a later point a submitter indicated that financial support was required
* How long ago did we accept a submission, we would like to send a reminder when a certain period passed
* ...

We think a modification date would be very useful. If there are other ways to achieve the same we would like to know that.

**Comment by rixx** [comment, marked as answer]
There's a log indicating *when* a proposal was changed, and by which user accoung, but not *what* was changed. The tracking issue is #400, any help is welcome!
